doc/contributing.asciidoc

```diff
@@ -85,6 +85,16 @@ git format-patch origin/master <1>
 <1> Replace `master` by the branch your work was based on, e.g.,
 `origin/develop`.
 
+Running Development Changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To run qutebrowser without installing it(useful for testing out small changes) run
```

Comment:
Can you add a space before the `(`, it seems a bit odd without it.

Also, this should say something about how this runs the git instance of qutebrowser, with modifications.

Reply:
Okay sure.